deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Files.Managed.Netcoreapp31.wxs

```diff
@@ -30,6 +35,11 @@
               Source="$(var.TracerHomeDirectory)\netcoreapp3.1\Sigil.dll"
               KeyPath="yes" Checksum="yes"/>
       </Component>
+      <Component Win64="$(var.Win64)">
+        <File Id="netcoreapp31_lzma.dll"
+              Source="$(var.TracerHomeDirectory)\netcoreapp3.1\lzma.dll"
```

Comment:
@tonyredondo: Do you know why this dependency got added? It is something we use for duck typing in specifically in .NET Core 3.1?

Reply:
No, that’s weird. Ducktyping only have two dependencies and for netstandard : https://github.com/DataDog/dd-trace-dotnet/blob/106a8e074944f2b2926a82554f0beeb39b69a924/src/Datadog.Trace.DuckTyping/Datadog.Trace.DuckTyping.csproj#L17

Reply:
When I `publish` the project on my machine the output has `runtimes/win7-x64/native/lzma.dll`.

Reply:
I’ll check but as you can see there is not a direct reference, something is kicking in for netcoreapp target.

Reply:
@lucaspimentel we can ignore this dependency. This is not used by the product and is only used transitively (through Sigil I believe) for a dotnet-zip tool. Aka it's not used

Reply:
Thanks. Removed it 080635dc.